titration/utils/constants.py

```diff
@@ -97,7 +101,7 @@
 DEFAULT_PH_REF_PH = 7.0
 # data paths
 DATA_PATH = "data/"
-volume_in_pump = 0  # keep track of solution in pump
+VOLUME_IN_PUMP = 0  # keep track of solution in pump
```

Comment:
While this may not change today, it should have some way of changing ... right?

Reply:
Yes, this is an issue that needs to be addressed. In the other PR it should be moved to the syringe pump file.

Reply:
But as a variable but not a constant